test_settings.py

```diff
@@ -38,8 +38,6 @@ def root(*args):
     root('event_routing_backends', 'conf', 'locale'),
 ]
 
-ROOT_URLCONF = 'event_routing_backends.urls'
```

Comment:
This module is deleted in this commit, had to remove this to run `makemigrations` command.

https://github.com/edx/event-routing-backends/commit/5531a185e49e356873b69cb0c58c4b0526c0318b#diff-f1b972652ffca6892272e92f8e805afa88f960473f084e6e07015dee212515d0